docs/src/tutorials/send-events.md

```diff
@@ -0,0 +1,58 @@
+# Thin Edge Event
+
+Events on thin-edge.io can be used to trigger signals when some event happens in the system.
+For example, a person entering a room or someone logging into a machine/website can all be represented as events.
+Events are stateless and hence are processed as and when they occur.
```

Comment:
It may be a bit out of context here, but 
There is a plan to group many events together and send them at once. Does that mean they occur at the same time or will they be raised in some time window ?.

Reply:
I wasn't aware of such a grouping requirement and I'm not quite sure how it will be done either. Will have to take it up when that story comes up for grooming.